### PR TITLE
linspace func in torchutil w/ endpoint option

### PR DESCRIPTION
**Pull request by maxsolomonhenry**
Pretty straightforward. Allows to keep the `np.linspace` behaviour we've been using and keeps the math nice. The only issue is the choice of function name. Currently it's `linspace` straight up.

`linspace_with_option_endpoint` is clear, but long. Another choice is `my_linspace` or something like that.

### CODE REVIEW

**Comment by turian** [comment]
Could you use linspace where I have the comment we want linspace = False?

Also, could you write a unit test in test_torchutil to fuzz test it's good?

**Comment by maxsolomonhenry** [comment]
I can but we have to merge this first -- there's no `tochmodule.py` in this branch

**Comment by turian** [comment]
@maxsolomonhenry I added an optional atol parameter, since relative tolerance is too sensitive for small values

**Comment by maxsolomonhenry** [comment]
Great, thanks. Looks like everything else is just issues with the main repo, which we'll have to clean up eventually in the torchmodule.py etc. So good to pull?

File: src/ddspdrum/torchmodule.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 from ddspdrum.defaults import SAMPLE_RATE
 from ddspdrum.modparameter import ModParameter
-from ddspdrum.torchutil import midi_to_hz, reverse_signal
+from ddspdrum.torchutil import midi_to_hz, reverse_signal, linspace
```
Comment (jorshi) [inline src/ddspdrum/torchmodule.py:15]: Linter is complaining about the ordering of imports, wants them to be alphabetical

Comment (maxsolomonhenry) [reply src/ddspdrum/torchmodule.py]: There's something a little S&M about this linting workflow.

**Pull request merged**